Dockerfile

```diff
@@ -21,12 +21,11 @@ RUN \
 
 # install ubooquity
  mkdir -p \
-	/opt/ubooquity \
-	/opt/ubooquity/fonts && \
+	/app/ubooquity && \
  curl -o \
  /tmp/ubooquity.zip -L \
 	"http://vaemendis.net/ubooquity/downloads/Ubooquity-${UBOOQUITY_VER}.zip" && \
- unzip /tmp/ubooquity.zip -d /opt/ubooquity && \
```

Comment:
why change from `/opt` ? isn't this the standard place to install software on a linux filesystem rather than making your own?